src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+                for(Task task: list) {
+                    counter = counter + 1;
+                }
+
+                objin.close();
+            } catch (Exception e) {
+                System.out.println("File is empty");
+            }
+        } else {
+            try {
+                save.createNewFile();
+            } catch (IOException e) {
+                System.out.println("Unable to create save file");
+            }
+        }
+
+        
+
+        while (!input.equals("bye")){
+            try{
+                if (input.equals("todo")){
+                    input = scan.nextLine();
+                    if (input.isEmpty() || input.isBlank()) { 
+                        throw new DukeException("OOPS!!! The description of a todo cannot be empty");
+                    }
+                    
+                    Todo t = new Todo(input); 
+                    list.add(t);
+                    counter = counter + 1;
+                    System.out.print("Got it. I've added this task:\n" + t.toString() + "\n" + "Now you have " + counter + " tasks in the list.\n");
+                    input = scan.next();
+                    continue;
+                }
+                
+                //deadline return book /by Sunday
+                else if (input.equals("deadline")){
+                    input = scan.nextLine();
+                    String[] arrOfStr = input.split("/by ");
+
+                    // input1 is the task
+                    String input1 = arrOfStr[0];
+                    // input2 is the deadline
+                    String input2 = arrOfStr[1];
+                    Deadline d = new Deadline(input1, input2); 
+                    list.add(d);
+                    counter = counter + 1;
+                    System.out.print("Got it. I've added this task:\n" + d.toString() + "\n" + "Now you have " + counter + " tasks in the list.\n");
+                    input = scan.next();
+                    continue;
+                }
+
+                //event project meeting /from Mon 2pm /to 4pm
+                else if (input.equals("event")){
+                    input = scan.nextLine();
+                    String[] arrOfStr = input.split("/from|/to");
+
+                    // input1 is the task
+                    String input1 = arrOfStr[0];
+                    // input2 is the start
+                    String input2 = arrOfStr[1];
+                    // input3 is the end
+                    String input3 = arrOfStr[2];
+
+                    Event e = new Event(input1, input2, input3); 
+                    list.add(e);
+                    counter = counter + 1;
+                    System.out.print("Got it. I've added this task:\n" + e.toString() + "\n" + "Now you have " + counter + " tasks in the list.\n");
+                    input = scan.next();
+                    continue;
+                }
+
+                else if (input.equals("mark")){
```

Comment:
Hey! I know you haven't gotten to the code-standard refinement stage yet, but I thought I'd just leave some comments on your code style (its a fast fix, don't worry).

Reply:
for your conditionals and everything that requires a () bracket, if there is curly braces after it, it would be best to leave a white space before the curly brace. eg.

```
else if (condition) {
    code...
}
```